src/main.js

```diff
@@ -74,6 +75,9 @@ export default class {
 				error_message: obj.message,
 				error_name: obj.name
 			};
+			if (obj.data) {
```

Comment:
Rather than just masking `data` should this loop through all other properties and mask them too? `data` is just what we have called it in `n-membership-sdk`.

Reply:
I was trying to prevent changing any existing functionality by using a specific property name.

Reply:
By adding a specific property in here this strays away from being a generic component. I would rather see this being an option passed in (if you need specificity) or done for all.